content/2015.12.loadtesting.with.oauth.rst

```diff
@@ -0,0 +1,214 @@
+Loadtesting a HTTP API which uses OAuth for authentication
+##########################################################
+
+:lang: en
+:date: 2015-12-02
+:url: loadtesting-a-http-api-which-uses-oauth-for-authentication
+:summary: How to loadtest an OAuth service relier?
+
+
+Why would you need to run loadtest?
+===================================
+
+After deploying an HTTP API service architecture you might be
+wondering how many users or requests your deploiement can take and
+what happens after it reaches that number.
+
+This is a great indicator to tell you to start scalling if you see
+that your production server is close to reach that number of requests
+per seconds.
+
+
+Why is it a problem to do so with OAuth?
+========================================
+
+Loadtesting with OAuth means having an OAuth account, often creating
```

Comment:
consider adding links to OAuth and Bearer Tokens, so the reader can understand what this is.